api/services/userService.js

```diff
@@ -0,0 +1,39 @@
+const bcrypt = require('bcrypt')
+
+const userDao = require('../models/userDao')
+
+const hashPassword = async (plainTextPassword) => {
+    
+        const SALT_ROUNDS = 12;
+        const salt = await bcrypt.genSalt(SALT_ROUNDS);
+        const hashedPassword = await bcrypt.hash(plainTextPassword, salt);
+        
+        return hashedPassword;
+}
+
+const signUp = async (name, email, password, phoneNumber) => {
+    const emailRegex    =/^[0-9a-zA-Z]([-_\.]?[0-9a-zA-Z])*@[0-9a-zA-z]([-_\.]?[0-9a-zA-Z])*\.[a-zA-Z]{2,3}$/
+	const passwordRegex =/^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!@#\$%\^&\*])(?=.{8,})/
```

Comment:
정규표현식도 하나의 문자열이기 때문에 변수 네이밍을 다음과 같이 하면 좋을 것 같습니다!
변수 수정에 따라 이후 부분도 수정해주세요.

```suggestion
    const EMAIL_REGEX    =/^[0-9a-zA-Z]([-_\.]?[0-9a-zA-Z])*@[0-9a-zA-z]([-_\.]?[0-9a-zA-Z])*\.[a-zA-Z]{2,3}$/
	const PASSWORD_REGEX =/^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!@#\$%\^&\*])(?=.{8,})/
```